test/test.js

```diff
@@ -0,0 +1,49 @@
+﻿const mocha = require('mocha');
+const chai = require('chai');
+const program = require('../words');
+const assert = chai.assert;
+chai.config.truncateThreshold = 0;
+describe('Test words function', () => {
+    it('should handle specified input', () => {
+        let testcase = program.words('olly olly in come free');
+        assert.deepEqual(testcase, { olly: 2, in: 1, come: 1, free: 1 });
+    });
+    it('should handle more complex input', () => {
+        let testcase = program.words('olly olly has come in the free has the free olly');
+        assert.deepEqual(testcase, { 'olly': 3, 'has': 2, 'come': 1, 'in': 1, 'the': 2, 'free': 2 });
+    });
+    it('should return an object', () => {
+        let testCase = typeof program.words('olly olly has come in the free has the free olly');
+        assert.deepEqual(testCase, 'object');
+    });
+    it('should handle multiple spaces', () => {
+        let testcase = program.words('olly  olly car    car');
+        assert.deepEqual(testcase, { 'olly': 2, 'car': 2 });
+    });
+    it('test that empty/space string return empty object', () => {
```

Comment:
The `it` block read out with the string you provided as description should flow as a normal English sentence. Picture this: `it test that empty/space string return empty object` and compare with: `it should return an empty object when an empty string is supplied as the input parameter/sentence`. What do you think?

Apply that to other test cases as well.

Reply:
Much better and eloquent, would makes changes immediately